content/en/observability_pipelines/processors/remap_ocsf/custom_mapping_configuration_format.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Custom Mapping Configuration Format
+disable_toc: false
+further_reading:
+- link: "logs/processing/pipelines"
+  tag: "Documentation"
+  text: "Log processing pipelines"
+---
+
+## Overview
+
+When you add a Remap OCSF processor with custom mapping, you can import your own OCSF mapping configuration. This document explains how to format your custom mapping file for the processor so that the processor can convert the events into OCSF format.
+
+## Mapping descriptor
+
+Your custom mapping file must be in the mapping descriptor format and can be in JSON or YAML. Out-of-the-box mappings use the filename `mapping.yaml`.
+
+The following is an example of the mapping descriptor:
+
+{{< tabs >}}
+{{% tab "YAML example" %}}
+
+```yaml
+version: 1
+metadata:
+  version: 1.1.0
+  class: Network Activity
+  profiles: [container, host, security_control]
+preprocess: []
+mapping: []
+
+```
+{{% /tab %}}
+{{% tab "JSON example" %}}
+
+```json
+{
+  "version": 1,
+  "metadata": {
+    "version": "1.1.0",
+    "class": "Network Activity",
+    "profiles": ["container", "host", "security_control"]
+  },
+  "preprocess": [],
+  "mapping": []
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The mapping descriptor has four main sections:
+
+| Name       | Required  | Description                                                                                                           |
+|------------|-----------|-----------------------------------------------------------------------------------------------------------------------|
+| `version`    | Yes       | Must be set to `1` to indicate the mapping descriptor format version.                                                 |
+| `metadata`   | Yes       | Contains a set of hard-coded description fields about the event class. See [Metadata section](#metadata-section) for details.                         |
+| `preprocess` | No        | An ordered list of preprocessing steps. The preprocessors reformat raw data from the sources so that the data can be converted to OCSF format based on `mapping`. Each preprocessor entry consists of a `function` and parameters associated with that function. See [Preprocessors](#preprocess-section) for more information. |
+| `mapping`    | Yes       | An ordered list of field-to-field assignments, where a `source` field is assigned to a `dest` field in the output OCSF event. See  for more information. Each [mapping](#mapping-section) may have a `conversion` specified by a [lookup table](#mapping-lookup-tables) or post-processing [mapping function](#mapping-functions). |
```

Comment:
I think there might be a missing link in the `mapping` description. (See for more information.)